### PR TITLE
Purge using namespace std from libyul/backends

### PR DESCRIPTION
**Pull request by junaire**
Related issue: https://github.com/ethereum/solidity/issues/14403

### CODE REVIEW

**Comment by junaire** [comment]
CC @nikola-matic

File: scripts/check_style.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ NAMESPACE_STD_FREE_FILES=(
     libsolidity/lsp/*
     libsolidity/parsing/*
     libsolutil/*
+    libyul/backends/*
```
Comment (nikola-matic) [inline scripts/check_style.sh]:
```suggestion
    libyul/backends/evm/*
```

Comment (junaire) [reply scripts/check_style.sh]: Done

**Comment by junaire** [comment]
gentle ping @nikola-matic

**Review by nikola-matic — APPROVED** [review]
I likely won't finish the optimizer PR today, just FYI.

**Comment by junaire** [comment]
> I likely won't finish the optimizer PR today, just FYI.

No worries, thanks for your review!

**Pull request merged**